Cargo.toml

```diff
@@ -1,18 +1,18 @@
 [workspace]
 
 members = [
-    "fvm_actor_utils",
+    "dispatch_examples/greeter",
     "frc42_dispatch",
     "frc42_dispatch/hasher",
     "frc42_dispatch/macros",
     "frc42_dispatch/macros/example",
-    "dispatch_examples/greeter",
-    "fvm_dispatch_tools",
     "frc46_token",
     "frcxx_nft",
+    "fvm_actor_utils",
+    "fvm_dispatch_tools",
     "testing/fil_token_integration",
```

Comment:
@abright to help make merge conflicts a bit clearer here I've sorted this alphabetically so that if packages are duplicated they'll show up obviously

Reply:
Nice. I remember using a vscode plugin to do stuff like this in the past, "Prettier TOML" or "Even Better TOML" or something like that.